apps/account-api/test/delegations.controller.e2e-spec.ts

```diff
@@ -20,20 +22,25 @@ let publicFollowsSchema: Schema | undefined;
 let privateFollowsSchema: Schema | undefined;
 let privateConnectionsSchema: Schema | undefined;
 let httpServer: any;
+let invalidMsaId: string;
+let msaNonProviderId: string;
 
 describe('Delegation Controller', () => {
   let app: INestApplication;
   let module: TestingModule;
 
   beforeAll(async () => {
-    ({ maxMsaId, provider, users } = await setupProviderAndUsers());
+    ({ maxMsaId, provider, revokedUsers: revokedUser, undelegatedUser, users } = await setupProviderAndUsers());
```

Comment:
a little confusing to have a plural -> singular. If a value is plural, I would assume it's an array.

Reply:
Good catch; that was a VSCode refactor anomaly. Should be singular; I'll fix.